src/Symfony/Component/HttpFoundation/Session/Storage/Handler/PdoSessionHandler.php

```diff
@@ -175,6 +182,88 @@ public function __construct(\PDO|string $pdoOrDsn = null, array $options = [])
         $this->ttl = $options['ttl'] ?? null;
     }
 
+    /**
+     * @throws Exception
+     */
+    private function isInListTablesWithoutFilter(AbstractSchemaManager $schemaManager): bool
+    {
+        $tablesName = $schemaManager->listTableNames();
+
+        if (\in_array($this->table, $tablesName)) {
+            return true;
+        }
+
+        return false;
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function configureSchema(Schema $schema, Connection $connection): void
+    {
+        $configuration = $connection->getConfiguration();
+
+        $schemaManager = method_exists(Connection::class, 'createSchemaManager')
+            ? $connection->createSchemaManager()
+            : $connection->getSchemaManager();
+
+        $configuration->setSchemaAssetsFilter(null);
```

Comment:
We need to figure something else than this call IMHO. Can't we change the default? Where is it set?

Reply:
Do you mean `setSchemaAssetsFilter` at the compiler pass level (WellKnowSchemaFilterPass and DbalSchemaFilterPass) ?

Reply:
Once `WellKnowSchemaFilterPass`will be removed, imho, we won't have any problems.

Reply:
`setSchemaAssetsFilter`to null is also manipulated by `Symfony\Component\Messenger\Bridge\Doctrine\Transport` even if after it's back to it's initial value. I can't do the same because of `createToSchema` in `Doctrine\Migrations\Generator\DiffGenerator`